jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -379,6 +417,15 @@ def vue_do_aper_phot(self, *args, **kwargs):
                                 f'{x:.4e} ({phot_table["aperture_sum_counts_err"][0]:.4e})'})
                 else:
                     tmp.append({'function': key, 'result': str(x)})
+            # Also display fit results
+            if fit_model is not None and isinstance(fit_model, Gaussian1D):

```

Comment:
The `isinstance` check is future-proofing because it is just a matter of time before someone asks for Moffat.